.travis.yml

```diff
@@ -1,37 +1,39 @@
 language: scala
-script:
-- "./bin/runci.sh $CI_TEST"
 jdk:
-- oraclejdk8
-env:
-  matrix:
-    - CI_TEST=scalafmt
-    - CI_TEST=ci-fast
+  - oraclejdk8
+
+stages:
+  - name: test
+  - name: release
+    if: (branch = master) AND (type = push)
+
+jobs:
+  include:
+    # default stage is test
+    - env: TEST="scalafmt"
+      script: ./bin/scalafmt --test
+    - env: TEST="languageserver/test"
+      script: sbt $TEST
+    - env: TEST="tests.ctags"
```

Comment:
I don't think we need to split the tests into different matrices yet. In #51 I added tests `tests.compiler`, which would not be caught by this. Full test suite still runs in <2-3 minutes.

Reply:
I suspect we'll want this later down the road however once we get heavier integration tests 👍

Reply:
OK. Addressed in cb64f24